docs/websocket/audio.md

```diff
@@ -20,3 +20,17 @@ name         | type  | description
 channels     | `int` | **Required** The number of channels.
 frame_rate   | `int` | **Required** The frame rate in Hertz.
 sample_width | `int` | **Required** The sample width in bits.
+
+
+## Self-describing formats
+
+A self-describing audio format has a file header with all required information.
+The [SoX documentation][1] elaborates on this in more technical details.
+
+Basically, any self-describing audio format *should* work, however only WAVE is
+supported.
```

Comment:
supported -> guaranteed to work

Reply:
I feel less comfortable to say it's guaranteed, it implies next level stability imo. Any reasons for this?